### PR TITLE
Fixes to etcd restore

### PR DESCRIPTION
**Pull request by jim-minter**
Fixing errors introduced into etcd restore by my previous commits.  Includes #975.

### CODE REVIEW

**Comment by mjudeikis** [comment]
/test e2e
might be related to https://github.com/openshift/openshift-azure/issues/973

**Comment by mjudeikis** [comment]
lgtm, will wait for other review.
I think biggest change here is VMSS model changes to include kuberenetes loadbalancer from the start.

File: pkg/arm/resources.go [hunk 1]
```diff
@@ -387,6 +494,9 @@ func Vmss(pc *api.PluginConfig, cs *api.OpenShiftManagedCluster, app *api.AgentP
 		},
 		VirtualMachineScaleSetProperties: &compute.VirtualMachineScaleSetProperties{
 			UpgradePolicy: &compute.UpgradePolicy{
+				AutoOSUpgradePolicy: &compute.AutoOSUpgradePolicy{
+					DisableAutoRollback: to.BoolPtr(false),
```
Comment (kargakis) [inline pkg/arm/resources.go]: In which case are we going to need this?

**Comment by kargakis** [comment]
one question about the autorollback feature but this can go in

/lgtm

File: pkg/arm/resources.go (hunk 1 above)
Comment (jim-minter) [reply pkg/arm/resources.go]: this is mirroring a change that the cloud provider appears to make to the VMSS by default - I put it in here to hopefully save the cloud provider finding a diff it needed to make and applying it.

Comment (jim-minter) [reply pkg/arm/resources.go]: (it should be a no-op from our perspective)

**Pull request merged**